core/dbt/materializations/incremental/microbatch.py

```diff
@@ -18,6 +18,7 @@ def __init__(
         is_incremental: bool,
         event_time_start: Optional[datetime],
         event_time_end: Optional[datetime],
+        batch_current_time: Optional[datetime] = None,
```

Comment:
Can this be `batch_invoked_at` for consistency?

Reply:
I'm down to rename the variable. However, I think `batch_invoked_at` would give the wrong impression. This datetime is not when the batch was actually invoked. The datetime will be before the batch is actually invoked, by how much is dependent on how long the project takes to run and when the microbatch model is being worked on within that.

Reply:
How about `batch_default_end_time`?

Reply:
or `default_end_time`? 🤔 Saying `batch` feels kinda redundant